README.md

```diff
@@ -22,3 +22,27 @@ Annotate log messages with named `SafeArg` and `UnsafeArg` as appropriate.  For
 
     // now
     log.info("Twisted the {} knob {} times", UnsafeArg.of("knobName", knobName), SafeArg.of("count", count));
+
+
+Preconditions
+=============
+Guava Preconditions equivalent which produces exceptions conforming to the SafeLoggable standard.
+
+Usage
+-----
+
+Add dependency to gradle:
+
+    compile "com.palantir.safe-logging:preconditions"
+    // optional test utilities
+    testCompile "com.palantir.safe-logging:preconditions-assertj"
+
+Annotate Preconditions error messages with named `SafeArg` and `UnsageArg` as appropriate.  For example:
+
+    // previously
+    Preconditions.checkArgument(twists > MIN_TWISTS, "%s twists less than minimum %s", twists, MINTWISTS);
+
+    // now
+    Preconditions.checkArgument(twists > MIN_TWISTS, "{} twists less than minimum {}", 
+            UnsafeArg.of("twists", twists), SafeArg.of("minimum", MIN_TWISTS));
```

Comment:
I'd prefer to find a more clear exampel for the unsafe param here as integers are normally considered safe.... Could we do somthing like "user entered string is too long" and then use unsafeArg for the string and safeArg for the length?

Also could we mention the imports so people understand they're two different `Preconditions` classes?

```
import com.google.common.base.Preconditions
...
import com.palantir.logsafe.Preconditions
```